docs/source/pages/About.rst

```diff
@@ -0,0 +1,28 @@
+.. role:: raw-html-m2r(raw)
```

Comment:
we will need to add a link to this page (I couldn't find any in the produced doc)

Reply:
Yes, I didn't want to put it in sidebar, do you see a place for it ?